pkg/motor/create_account.go

```diff
@@ -152,18 +164,13 @@ func updateWhoIs(m *motorNodeImpl) (*rt.MsgUpdateWhoIsResponse, error) {
 	return cwir, nil
 }
 
-func pskEncrypt(shard []byte) ([]byte, []byte, error) {
-	key, err := mpc.NewAesKey()
-	if err != nil {
-		return nil, nil, err
-	}
-
+func pskEncrypt(shard, key []byte) ([]byte, error) {
 	cipherShard, err := mpc.AesEncryptWithKey(key, shard)
```

Comment:
This makes me a bit suspicious, but its not part of the changeset

Reply:
Will bring this up in standup tomorrow!